Src/AutoFixture.NUnit3.UnitTest/InlineAutoDataAttributeTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class InlineAutoDataAttributeTest
+    {
+        [Test]
+        public void InlineAutoDataInheritsAutoData()
+        {
+            Assert.That(new InlineAutoDataAttribute(), 
+                Is.AssignableTo(typeof(AutoDataAttribute)));
+        }
+
+        [Test]
+        public void InlineAutoDataCanBeExtendedWithImplementationOfIFixture()
+        {
+            var extended = new InlineAutoDataAttributeStub();
+
+            Assert.That(extended, Is.AssignableTo(typeof(InlineAutoDataAttribute)));
```

Comment:
What does it mean if `extended` which, is an instance of the SUT, is assignable to the SUT itself?